### ISSUE TITLE
Fail to build on linux mint

### ISSUE THREAD

**Issue by ColumPaget** [linux-pam/linux-pam#766]
Hi, in order to test reproducibility with the other issue I've posted here, I've tried building on linux mint, and found something else. 

This is with a tarball taken from 'releases' in this github. Compiler is gcc "(Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0".  configure is just "configure --prefix=/". This is an out-of-tree build in a separate build directory.

Build fails at:

../../../Linux-PAM-1.6.0/modules/pam_namespace/pam_namespace.c: In function ‘process_line’:
../../../Linux-PAM-1.6.0/modules/pam_namespace/pam_namespace.c:649:41: error: ‘SIZE_MAX’ undeclared (first use in this function)
  649 |         if (count > UINT_MAX || count > SIZE_MAX / sizeof(uid_t)) {
      |                                         ^~~~~~~~
../../../Linux-PAM-1.6.0/modules/pam_namespace/pam_namespace.c:41:1: note: ‘SIZE_MAX’ is defined in header ‘<stdint.h>’; did you forget to ‘#include <stdint.h>’?
   40 | #include "argv_parse.h"
  +++ |+#include <stdint.h>
   41 | 
../../../Linux-PAM-1.6.0/modules/pam_namespace/pam_namespace.c:649:41: note: each undeclared identifier is reported only once for each function it appears in
  649 |         if (count > UINT_MAX || count > SIZE_MAX / sizeof(uid_t)) {
      |                                         ^~~~~~~~
make[3]: *** [Makefile:815: pam_namespace.lo] Error 1
make[3]: Leaving directory '/home/colum/tmp/modules/pam_namespace'

Adding #include <stdint.h> to pam_namespace as suggested solves the issue, although I'm not sure that's the "correct" thing to do.

After that build succeeds, but the same issue is seen with libpam_misc at install time, as raised in my other issue.

**Comment by ldv-alt** [comment]
This is a duplicate of #733.

**Comment by ColumPaget** [comment]
Okay, closing this issue then.
